internal/pkg/term/selector/selector.go

```diff
@@ -73,6 +86,12 @@ type DeploySelect struct {
 	env            string
 }
 
+// Ec2Select is a selector for Ec2 resources.
```

Comment:
can we move the ec2 related selector code to its own file? `selector/ec2.go`

Reply:
yeah good idea but i think we then would be better to split up selector completely into `config.go` and `deploy.go`.